psqlextra/partitioning/plan.py

```diff
@@ -42,6 +44,14 @@ def apply(self, using: Optional[str]) -> None:
                         comment=AUTO_PARTITIONED_COMMENT,
                     )
 
+                for partition in self.detachements:
+                    partition.detach(self.config.model, schema_editor, concurrently=False)
```

Comment:
Any reason we want to detach partition in the same transaction as creation of partitions? In case of failure in detach, we won't create new partitions, isn't it?